modules/tinymce/src/core/main/ts/api/util/EventDispatcher.ts

```diff
@@ -139,7 +139,7 @@ class EventDispatcher<T> {
    * @param {Object?} args Event arguments.
    * @return {Object} Event args instance passed in.
    * @example
-   * instance.fire('event', {...});
+   * editor.fire('event', {...});
```

Comment:
Is this correct? As this won't be the editor for event dispatcher (it will be for `EditorObservable`), it'll be the event dispatcher instance. So I believe this part was correct originally.